doc/nrf/gs_ins_windows.rst

```diff
@@ -83,20 +83,22 @@ The |NCS| consists of the following repositories:
 Every |NCS| release consists of a combination of these repositories at different revisions.
 
 .. note::
-   The latest state of development is on the master branches of the repositories.
-   Note that this might not be a stable state.
+   The latest state of development is on the master branch of the fw-nrfconnect-nrf repository.
    To ensure a usable state, the fw-nrfconnect-nrf repository defines the compatible states of the other repositories.
-   However, this state is not necessarily tested.
+   However, this state is not necessarily tested. Releases and tags however do guarantee a higher quality level.
```

Comment:
Every sentence should start on a new line (see style guide).
Suggestion for the second sentence:

For a higher quality level, check out a tagged release.